stencil/src/components/pulumi-api-doc-filterable-nav/pulumi-api-doc-filterable-nav.tsx

```diff
@@ -31,19 +31,34 @@ export class PulumiApiDocFilterableNav {
     }
 
     componentWillLoad() {
-        // Parse the JSON that's passed through as a prop.
-        this.parsedNodes = JSON.parse(this.nodes);
-        // Before the user interacts with the filter, the nodes to render should be the full nav tree.
-        this.currentlyRenderedNodes = this.parsedNodes;
+        this.loadNavigationData();
```

Comment:
I assume `componentWillLoad` cannot be marked as `async`? Just for completeness sake, can you please subscribe to this Promise result with a noop `then()`? That ensures we don't have a dangling promise.